fmriprep_rodents/workflows/bold/registration.py

```diff
@@ -717,7 +718,8 @@ def init_fsl_bbr_wf(use_bbr, bold2t1w_dof, bold2t1w_init, sloppy=False, name='fs
     else:
         # Should mostly be hit while building docs
         LOGGER.warning("FSLDIR unset - using packaged BBR schedule")
-        flt_bbr.inputs.schedule = pkgr.resource_filename('fmriprep_rodents', 'data/flirtsch/bbr.sch')
+        flt_bbr.inputs.schedule = pkgr.resource_filename(
+            'fmriprep_rodents', 'data/flirtsch/bbr.sch')
```

Comment:
As I said before, this makes me a bit worried that we are running bbr and we shouldn't